services/task_store/service.go

```diff
@@ -459,15 +460,49 @@ func (ts *Service) handleTask(w http.ResponseWriter, r *http.Request) {
 		httpd.HttpError(w, fmt.Sprintf("invalid dot-view parameter %q", dotView), true, http.StatusBadRequest)
 		return
 	}
+
 	tmID := r.URL.Query().Get("replay-id")
 	if tmID == "" {
 		tmID = kapacitor.MainTaskMaster
 	}
+
+	finished := r.URL.Query().Get("finished")
```

Comment:
Why are we making a request to the `/tasks` endpoint to get replay stats? Why not just make a request to `/replays?finished=true` and then return the stats there? 

It means the external system needs to remember which replay-ids map to which tasks but that seems reasonable since the external system made the original request to create a replay with the specified task.

Also not all replays are associated with a task, which is why Kapacitor doesn't store the association itself.

Reply:
Yes, it makes sense. I will move the handler code as is under the `replay` package.